### DISCUSSION TITLE
[Question][Hw6] Where can I download assigment 6 problems?

### DISCUSSION THREAD

**Discussion by Jaegoomon** [prosyslab-classroom/cs348-information-security#637] [Q&A]
Where can I download assigment 6 problems?

It seems there isn't in KLMS...

**Comment by mickowale** [comment]
https://www.dropbox.com/s/ppxnrpzcse7w11r/CS448_21S_Assignment_6.pdf?dl=0

**Comment by minsukk-at-kaist-ac-kr** [comment]
I sent out an email several days ago...

--

Dear CS448 folks,



The very last assignment of CS448 is finally here for you: https://www.dropbox.com/s/ppxnrpzcse7w11r/CS448_21S_Assignment_6.pdf?dl=0



Please note that the deadline is in a week time and make sure you check your remaining grace days before planning your submission.



We will soon upload the remaining grace days in the KLMS system so that you can double check. 



Also, make sure you read the fine print on the first page, especially the new length policy. 



Good luck!



Thanks,

Min Suk

**Comment by Jaegoomon** [comment]
Oh I'm so sorry..

Thank you professor

**Comment by minsukk-at-kaist-ac-kr** [comment]
:-)
